src/utils/keyword_extractor.py

```
@@ -0,0 +1,1026 @@
+"""
```

Pylint (3.3.7)

[convention] Too many lines in module (1026/1000)

(C0302)